histograms.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jan 31 13:12:27 2019
+
+@author: courtney
+"""
+
+'''
+Creating histogram images
+'''
+
+import numpy as np
+import matplotlib.pyplot as plt
+import os
+
+os.chdir('/home/courtney/Desktop/practice/chips')
+
+ubins = []
+bins = []
+
+uim01 = plt.imread('glacier003.2018-08-01.tif')
+uim07 = plt.imread('glacier003.2018-08-07.tif')
+uim13 = plt.imread('glacier003.2018-08-13.tif')
+uim19 = plt.imread('glacier003.2018-08-19.tif')
+uim25 = plt.imread('glacier003.2018-08-25.tif')
+
+'''ubins = [0 to 255]'''
```

Comment:
Is this a comment? Line comments are prefaced with # .. I think ''' is only used for multi-line strings (including the docstrings in functions - Python knows not to interpret that as code, but if there's '''stuff''' just floating in the script it might cause problems)

Reply:
Yeah it was supposed to be a comment, I'll change it back to a #